### PR TITLE
Scoring period 3

### PR DESCRIPTION
**Pull request by bwhm**

### CODE REVIEW

File: scoring-periods/3.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+## `Scoring Period 3`
+### _15/03/21 - 28/03/21_
+<br>
+
+
+#### Announcement
+We are happy to announce the third "official" Scoring Period ([Scoring Period #0](/scoring-periods/0.md) was also created in order to recognize existing contributors to the project).
```
Comment (blrhc) [inline scoring-periods/3.md:7]:
```suggestion
```

File: scoring-periods/3.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+After the overwhelming amount of submissions for [Scoring Period #1](/scoring-periods/1.md), it was a lot more time consuming than anticipated to grade it, causing some grading delays and errors on our end.
```
Comment (blrhc) [inline scoring-periods/3.md:9]:
```suggestion
Due to the overwhelming number of submissions received for [Scoring Period #1](/scoring-periods/1.md), it ended up being significantly more time consuming than we had previously anticipated to grade, causing some delays and issues on our end.
```

File: scoring-periods/3.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+We will fix, and update these in the coming days, and start grading the submissions for [Scoring Period #2](/scoring-periods/2.md) in the the 17th of March.
```
Comment (blrhc) [inline scoring-periods/3.md:11]:
```suggestion
Needless to say, we plan to start grading the submissions for [Scoring Period #2](/scoring-periods/2.md) over the next week, and we hope we will be able to publish the results for this round much earlier now that we have learned some important lessons from the first round.
```

File: scoring-periods/3.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+
+Again, here are some pointers to submitting based on what we learned during grading of Scoring Periods 0 and 1!
+
+1. There is no need to submit your summary before the end of a Scoring Period!
+  - It appears lots of people believe that it's a "race", and that you have to submit early and often.
+  - A summary should be the report of all your activities during the entire scoring period. In other words, unless you _know_ you will not make any further contributions, wait until the Scoring Period has ended.
+  - Starting from this Scoring Period (ie. 3), we will reserve the right to only grade one summary per person per Scoring Period.
+    - As this our fault, we will of course not "punish" anyone for this for periods 1 or 2
```
Comment (blrhc) [inline scoring-periods/3.md:19]:
```suggestion
    - As the rules on this were not initially clear, we will of course not penalize anyone for having done this for periods 1 or 2
```

File: scoring-periods/3.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+After the overwhelming amount of submissions for [Scoring Period #1](/scoring-periods/1.md), it was a lot more time consuming than anticipated to grade it, causing some grading delays and errors on our end.
+
+We will fix, and update these in the coming days, and start grading the submissions for [Scoring Period #2](/scoring-periods/2.md) in the the 17th of March.
+
+Again, here are some pointers to submitting based on what we learned during grading of Scoring Periods 0 and 1!
+
+1. There is no need to submit your summary before the end of a Scoring Period!
+  - It appears lots of people believe that it's a "race", and that you have to submit early and often.
+  - A summary should be the report of all your activities during the entire scoring period. In other words, unless you _know_ you will not make any further contributions, wait until the Scoring Period has ended.
```
Comment (blrhc) [inline scoring-periods/3.md:17]:
```suggestion
  - A summary should be the report of all your activities during the entire Scoring Period. In other words, unless you _know_ you will not make any further contributions, wait until the Scoring Period has ended.
  - There is a short "grace period" of about 4-5 days after the Scoring Period has ended where you can submit your summary, so as long as you don't wait too long too write your report, you should be fine!
```

File: scoring-periods/3.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+  - Starting from this Scoring Period (ie. 3), we will reserve the right to only grade one summary per person per Scoring Period.
+    - As this our fault, we will of course not "punish" anyone for this for periods 1 or 2
+2. The program is not ending anytime soon!
+  - From interactions with the community, we have seen some people be of the impression that this is the one and only time to act in order to get points.
```
Comment (blrhc) [inline scoring-periods/3.md:21]:
```suggestion
  - From interactions with the community, we have seen some people be of the impression that there is a great rush to accumulate points.
```

File: scoring-periods/3.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+  - In fact, we plan to keep the project running until we are close to mainnet launch which is least 9 months away.
+  - If these plans change, we will notify the community in advance.
+3. We _do_ have a referral program!
+  - This has now been highlighted [here](/README.md#referral-program)
+  - Refer your friends, and start earning points when they submit their summaries!
+4. We have now made it so that just running a node will give (almost) as many points!
+  - Due to the competitiveness for the Validator spots making it hard for new users to get a spot, we needed to allow users to earn points simply for running (a) node(s).
```
Comment (blrhc) [inline scoring-periods/3.md:28]:
```suggestion
  - Due to competitiveness for the Validator spots making it hard for new users to get a spot, we felt that it was very important to allow new users (without a substantial tJOY balance) to earn points simply for running nodes, as this requires almost the same level of effort as running a Validator.
```

File: README.md [hunk 8]
```diff
@@ -91,9 +91,9 @@ The full rules of the Founding Member Program can be found [here](/RULES.md).
 
 ### Current Period
 
-The current [Scoring Period #2](/scoring-periods/2.md) started on the 1st of March, and ends on the 14th of March.
+The current [Scoring Period #3](/scoring-periods/3.md) started on the 15th of March, and ends on the 28th of March.
 
-Note that we are still accepting summaries from [Scoring Period #0](/scoring-periods/0.md) and [Scoring Period #1](/scoring-periods/1.md).
+Note that we are still accepting summaries from Scoring Periods [#0](/scoring-periods/0.md), [#1](/scoring-periods/1.md) and [#2](/scoring-periods/2.md).
```
Comment (blrhc) [inline README.md:96]:
```suggestion
Note that we are still accepting summaries from Scoring Periods [#0](/scoring-periods/0.md), [#1](/scoring-periods/1.md) and [#2](/scoring-periods/2.md), but only where a report for these periods has not previously been submitted.
```

**Review by blrhc — APPROVED** [review]
Approving if the suggested changes are implemented.
I am also going to do another PR since there are a couple of issues I would like to fix which are not optimally done through this PR.

**Comment by bwhm** [comment]
> Approving if the suggested changes are implemented.
> I am also going to do another PR since there are a couple of issues I would like to fix which are not optimally done through this PR.

Thanks, implemented all!

Please merge :)

**Pull request merged**